src/rust/cxx-integration/BUILD

```diff
@@ -0,0 +1,21 @@
+load("//:build/wd_rust_crate.bzl", "rust_cxx_include", "wd_rust_crate")
+
+wd_rust_crate(
+    name = "cxx-integration",
+    cxx_bridge_src = "lib.rs",
+    visibility = ["//visibility:public"],
+    deps = [
+        "@capnp-cpp//src/kj",
+        # "//src/rust/process-tracing",
+        # "//src/rust/safe-libc",
+        # "//src/rust/version",
```

Comment:
Nit: It is doubtful if we'll need a version field as used internally anytime soon for workerd